config/initializers/audited_hacks.rb

```diff
@@ -167,6 +174,9 @@ def provider_id_for_audits
 end
 
 ActiveSupport.on_load(:active_record) do
+  # Unlike above, this disables auditing globally
+  Audited.auditing_enabled = false if Rails.env.test?
+
   # we want to audit created_at field
   Audited.ignored_attributes = %w(lock_version updated_at created_on updated_on)
 
```

Comment:
I don't completely follow the changes in this file.  Can you add some quick notes about what you're doing here?  :)

Reply:
Adding some notes to the commit message, thanks!